css/styles.css

```diff
@@ -0,0 +1,18876 @@
+.DateRangePicker {
+  position: relative;
+  display: inline-block
+}
+.DateRangePicker_picker {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker__rtl {
+  direction: rtl
+}
+.DateRangePicker_picker__directionLeft {
+  left: 0px
```

Comment:
also that zero values have units

Reply:
again this is due partly to the autogeneration of this file and i need to... probably clean it up a bit more/read more into clean-css options.